dev-python/urwidtrees/urwidtrees-1.0.2-r1.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
```

Comment:
Could you bump it to EAPI 7 while at it?

Reply:
yes fixed